ramalama/sandbox.py

```
@@ -201,6 +234,137 @@ def add_env_options(self, args: OpenCodeArgsType) -> None:
+        """Remove temporary config file and stop gateway container."""
+        if hasattr(self, "config_file_path") and self.config_file_path:
+            try:
+                os.unlink(self.config_file_path)
+            except OSError:
+                pass
+        if hasattr(self, "_gateway_name"):
+            try:
+                self.engine.args.ignore = True  # type: ignore[attr-defined]
+                stop_container(self.engine.args, self._gateway_name, remove=True)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)